L'exécution de votre programme a déclenché une exception java.lang.ArrayIndexOutOfBoundsException : 12
 Merci de corriger votre code.
   at plm.runtime5.LangtonsAnt$$anonfun$step$1.apply$mcVI$sp (plm.runtime5.LangtonsAnt:14)
   at plm.runtime5.LangtonsAnt$$anonfun$step$1.apply (plm.runtime5.LangtonsAnt:13)
   at plm.runtime5.LangtonsAnt$$anonfun$step$1.apply (plm.runtime5.LangtonsAnt:13)
   at scala.collection.immutable.Range.foreach (Range.scala:166)
   at plm.runtime5.LangtonsAnt.step (plm.runtime5.LangtonsAnt:13)
   at plm.runtime5.LangtonsAnt$$anonfun$run$2.apply$mcVI$sp (plm.runtime5.LangtonsAnt:45)
   at plm.runtime5.LangtonsAnt$$anonfun$run$2.apply (plm.runtime5.LangtonsAnt:43)
   at plm.runtime5.LangtonsAnt$$anonfun$run$2.apply (plm.runtime5.LangtonsAnt:43)
   at scala.collection.immutable.Range.foreach (Range.scala:166)
   at plm.runtime5.LangtonsAnt.run (plm.runtime5.LangtonsAnt:43)
   at plm.core.lang.JVMCompiledLang.runEntity (null:-1)
   at plm.universe.World$1.run (null:-1)
   at java.lang.Thread.run (Thread.java:745)
